vuepress/docs/next/tutorials/solution/customer-portal.md

```diff
@@ -25,11 +25,15 @@ This tutorial covers:
 * Use the Entando CLI command `ent check-env develop` to verify all dependencies.
 * A Jira Service Management account.
 
-### Automatic Install with Entando App Builder
+### Auto Install via the Entando Hub
 Install the Customer Portal in two steps by integrating the Entando Hub into your App Builder. 
-1. From the left sidebar of your App Builder, go to the `Repository`.  Select `Entando Hub` at the top right side of the page if it has already been configured. If not, enter `Entando Hub` and the API URL to initialize the Hub. It will now be available from this page at any time.
+1. Log into your `App Builder` → `Repository` → `Select Registry` kebob buttons 
```

Comment:
kebob buttons? Have we used that term before?

Reply:
i just think like chicken kebobs :D

Reply:
i have never heard of this and was totally confused by it